src/parser/jobs/sch/modules/ScholarCooldownDowntime.tsx

```diff
@@ -0,0 +1,23 @@
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
+import React, {Fragment} from 'react'
+
+export default class ScholarCooldownDowntime extends CooldownDowntime {
+	override trackedCds = [
+		{
+			cooldowns: [this.data.actions.CHAIN_STRATAGEM],
+			allowedAverageDowntime: 7500,
+			firstUseOffset: 10000,
+		},
+		{
+			cooldowns: [this.data.actions.AETHERFLOW],
+			allowedAverageDowntime: 5000,
+			firstUseOffset: 10000,
+		},
+	]
+	override checklistDescription = <Fragment>
+		<div><Trans id="sch.cooldown.chainstrat.description">Try to use <ActionLink {...this.data.actions.CHAIN_STRATAGEM}/> on cooldown throughout the fight, particularly when your team has other buffs up for maximum effect.</Trans></div>
+		<div><Trans id="sch.cooldown.aetherflow.description">Try to use <ActionLink {...this.data.actions.AETHERFLOW}/> on cooldown throughout the fight. If you have extra Aetherflow charges, cast Energy Drain to do additional damage before casting <ActionLink {...this.data.actions.AETHERFLOW}/>.</Trans></div>
+	</Fragment>
```

Comment:
Any reason this needs a div or explicit Fragment? Compared to say,
```suggestion
	override checklistDescription = <>
		<Trans id="sch.cooldown.chainstrat.description">Try to use <ActionLink action="CHAIN_STRATAGEM"/> on cooldown throughout the fight, particularly when your team has other buffs up for maximum effect.</Trans>
		<Trans id="sch.cooldown.aetherflow.description">Try to use <ActionLink action="AETHERFLOW"/> on cooldown throughout the fight. If you have excess charges, cast <ActionLink action="SCH_ENERGY_DRAIN"/> to do additional damage before casting <ActionLink showIcon={false} action="AETHERFLOW"/>.</Trans>
	</>
```

I'm not sure this is actually a good idea tho, the double message is awkward UX imo.

Reply:
It looked awkward all as one paragraph to me. So I added the explicit divs to force a line break. If there's a way to do that more idomatically in react I'm game. But to your other point maybe it is better to just change it to a generic message for the cooldown section, and then maybe add more specific messages as conditional suggestions.

Reply:
I don't see any suggestions but that sounds like a more reasonable approach.

Reply:
After looking at other CDDT modules it seems standard to not include suggestions in the CDDT module. So I just removed the custom checklistDescription altogether.